com.unity.netcode.gameobjects/Runtime/Messaging/Messages/ParentSyncMessage.cs

```diff
@@ -1,29 +1,49 @@
+using UnityEngine;
+
 namespace Unity.Netcode
 {
     internal struct ParentSyncMessage : INetworkMessage
     {
         public ulong NetworkObjectId;
 
-        public bool IsReparented;
+        public bool WorldPositionStays;
 
         //If(Metadata.IsReparented)
         public bool IsLatestParentSet;
 
         //If(IsLatestParentSet)
         public ulong? LatestParent;
 
+        // Is set when the parent should be removed (similar to IsReparented functionality but only for removing the parent)
+        public bool RemoveParent;
+
+        // These additional properties are used to synchronize clients with the current position
+        // , rotation, and scale after parenting/de-parenting (world/local space relative). This
```

Comment:
What's the leading " , " before rotation? Can we send the comma one line up, right after "position" ?

Reply:
yeah, that does look funny... it is the 1 whitespace requirement after the start of a comment line that makes it look odd.
It should be:  position, rotation, and scale
But...when you put that comma on the next line it does seem visually perplexing now that you brought it to my attention.
I can move the comma up to the previous line.